lib/al/Library/System/SystemKit.h

```diff
@@ -29,8 +27,4 @@ class SystemKit {
     al::ResourceSystem* getResourceSystem() { return mResourceSystem; }
     al::SaveDataDirector* getSaveDataDirector() { return mSaveDataDirector; }
 };
-}  // namespace al
-
-class alProjectInterface {
-    static al::SystemKit* getSystemKit();
-};
```

Comment:
Why remove this?

Reply:
Moved to System/ProjectInterface.h as that's where this function is located in the binary.